R/deprecated.R

```diff
@@ -1,349 +1,3 @@
-#' Deprecated `TealData` class and related functions
```

Comment:
Also removed in `teal` https://github.com/insightsengineering/teal/pull/1406/files